tools/ab_test.py

```diff
@@ -78,16 +78,12 @@ def extract_dimensions(emf):
     return {key: emf[key] for key in emf if key in dimension_list}
 
 
-def reemit_emf_and_get_data(log_entry: str, revision: str):
+def process_log_entry(emf: dict):
     """Parses the given EMF log entry, and reemits it, overwriting the attached "git_commit_id" field
     with the given revision
```

Comment:
I guess the comment does not match the code now?

Reply:
ahh, true, fixed!